packages/compiler-cli/ngcc/test/rendering/umd_rendering_formatter_spec.ts

```diff
@@ -235,10 +221,34 @@ exports.BadIife = BadIife;
                 file);
             expect(output.toString())
                 .toContain(
-                    `typeof exports === 'object' && typeof module !== 'undefined' ?\n` +
-                    `    factory(require('@angular/core'),require('@angular/common'),exports, require('some-side-effect'), require('/local-dep'), require('@angular/core')) :`);
+                    wrapperFunctionFormat === WrapperFunctionFormat.Rollup ?
+                        `typeof exports === 'object' && typeof module !== 'undefined' ?\n` +
+                            `    factory(require('@angular/core'),require('@angular/common'),exports, require('some-side-effect'), require('/local-dep'), require('@angular/core')) :` :
+                        `if (typeof exports === 'object' && typeof module === 'object')\n` +
+                            `    module.exports = factory(require('@angular/core'),require('@angular/common'),require('some-side-effect'), require('/local-dep'), require('@angular/core'));`);
           });
 
+          if (wrapperFunctionFormat === WrapperFunctionFormat.Webpack) {
+            // The CommonJS (vs CommonJS2) format only applies to the Webpack format.
+            it('should append the given imports into the CommonJS2 factory call', () => {
```

Comment:
I am confused here. The condition enables this test only for Webpack wrappers and the test description tells me that that uses CommonJS2, but I read the comment as the exact opposite (where Webpack only uses CJS)

Reply:
I think this should be 

```suggestion
          if (wrapperFunctionFormat === WrapperFunctionFormat.Webpack) {
            // The CommonJS (vs CommonJS2) format only applies to the Webpack format.
            it('should append the given imports into the CommonJS factory call', () => {
```

We are checking here that in webpack there is also CommonJS (v1) as well as CommonJS2.
The test above this already tests the CommonJS2 format for both rollup and webpack.
Right?

Reply:
Ooops, yes, it should say `CommonJS` in the test description 😇